app/views/pages/home.html.erb

```diff
@@ -1,10 +1,12 @@
 <main role="main" class="container">
   <div class="row justify-content-center">
-    <h1>Welcome to Mayflower Community Kitchen!</h1>
+    <h1><MUAHAHAH JENKINS DEPLOYED THIS WOOT WOOT</h1>
   </div>
   <div class="row justify-content-center">
     <div class="col-md">
-      <p>Welcome to our temporary homepage!</p>
+
+      <p>Moreeeese BLAasdasdaAHH</p>
+
```

Comment:
Lolll.. are those just for testing purpose or? We do want the title to be "Welcome to Mayflower Community Kitchen" instead of  ..., right?